demo/d2l-user.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta charset="UTF-8">
+		<style>
+			body {
+				margin: 0;
+			}
+		</style>
+		<script type="module">
+			import '@brightspace-ui/core/components/demo/demo-page.js';
+			import '@brightspace-ui/core/components/typography/typography.js';
+			import '../components/d2l-user.js';
+		</script>
```

Comment:
```suggestion
		<link rel="stylesheet" href="/node_modules/@brightspace-ui/core/components/demo/styles.css" type="text/css">
		<script type="module">
			import '@brightspace-ui/core/components/demo/demo-page.js';
			import '../components/d2l-user.js';
		</script>
```
Does this look the same as what you have?

Reply:
Oh, awesome! Didn't know about this 😄